wger/weight/templates/overview.html

```diff
@@ -110,6 +110,12 @@
                 {% trans "Import from spreadsheet" %}
             </a>
         </li>
+        <li>
+            <a href="{% url 'weight:fitbit' %}" {% auto_link_css flavour %}>
```

Comment:
@Manorlds-Eaglespark I have tested out the integration and works as expected and described in your PR description. As a user, I can import my data from `Fitbit` into `wger`